test/slack_test.rb

```diff
@@ -128,6 +128,20 @@ def test_v2_custom_alerts
       assert_not_nil(attachment["fields"])
       assert_equal(1, attachment["fields"].length)
       assert_not_nil(attachment["mrkdwn_in"])
+      assert (not attachment["pretext"].include?('This is a test message'))
```

Comment:
Here and elsewhere, perhaps "This is a test alert notification" might be more descriptive?